vignettes/custom_priors.Rmd

```diff
@@ -0,0 +1,156 @@
+enw_reference_prior <- data.frame(
+  variable = c(
+    "refp_mean_int", "refp_sd_int", "refp_mean_beta_sd", "refp_sd_beta_sd",
+    "refnp_int", "refnp_beta_sd"
+  ),
+  description = c(
+    "Log mean intercept for parametric reference date delay",
+    "Log standard deviation for the parametric reference date delay",
+    "Standard deviation of scaled pooled parametric mean effects",
+    "Standard deviation of scaled pooled parametric sd effects",
+    "Intercept for non-parametric reference date delay",
+    "Standard deviation of scaled pooled non-parametric effects"
+  ),
+  distribution = c(
+    "Normal", rep("Zero truncated normal", 3),
+    "Normal", "Zero truncated normal"
+  ),
+  mean = c(1, 0.5, 0, 0, 0, 0),
+  sd = 1,
+  stringsAsFactors = FALSE
+)
+knitr::kable(enw_reference_prior)
+```
+## Reporting date module
+
+The reporting date model module has a single prior.
+```{r reporting_prior, echo = FALSE}
+enw_report_prior <- data.frame(
+  variable = "rep_beta_sd",
+  description = "Standard deviation of scaled pooled report date effects",
+  distribution = "Zero truncated normal",
+  mean = 0,
+  sd = 1,
+  stringsAsFactors = FALSE
+)
+knitr::kable(enw_report_prior)
+```
+
+
+## Expectation module
+
+The expectation module has three priors.
+
+```{r expectation_prior, echo = FALSE}
+enw_expectation_prior <- data.frame(
+    variable = c(
+      "expr_r_int", "expr_beta_sd", "expr_lelatent_int", "expl_beta_sd"
+    ),
+    description = c(
+      "Intercept of the log growth rate",
+      "Standard deviation of scaled pooled log growth rate effects",
+      "Intercept for initial log observations (ordered by group and then time)",
+      "Standard deviation of scaled pooled log growth rate effects"
+    ),
+    distribution = c(
+      "Normal", "Zero truncated normal", "Normal", "Zero truncated normal"
+    ),
+    mean = c(0, 0, "I dont understand", 0),
+    sd = c(0.2, 1, 1, 1),
+  stringsAsFactors = FALSE
+  )
+knitr::kable(enw_expectation_prior)
+```
+
+# Missing reference data model module
+
+The missing reference data module has a two priors.
+
+```{r missing_prior, echo = FALSE}
+enw_missing_prior <- data.table::data.table(
+  variable = c("miss_int", "miss_beta_sd"),
+  description = c(
+    "Intercept on the logit scale for the proportion missing reference dates",
+    "Standard deviation of scaled pooled logit missing reference date effects"
+  ),
+  distribution = c("Normal", "Zero truncated normal"),
+  mean = c(0, 0),
+  sd = c(1, 1),
+  stringsAsFactors = FALSE
+)
+knitr::kable(enw_missing_prior)
+```
+
+# Observation module
+
+The observation module has a single prior.
+
+```{r observation_prior, echo = FALSE}
+enw_obs_prior <- data.frame(
+  variable = "sqrt_phi",
+  description = "One over the square root of the reporting overdispersion",
+  distribution = "Zero truncated normal",
+  mean = 0,
+  sd = 1,
+  stringsAsFactors = FALSE
+)
+knitr::kable(enw_obs_prior)
+```
+
+## Customising priors
```

Comment:
it might be nice to pivot this vignette a bit and show 1. How to directly specify priors, 2. How to reuse posteriors from a fit model. 

If we thought about it like and using above comment about looking at the content of the modules we could do all of this based on an actual example?

Reply:
> If we thought about it like and using above comment about looking at the content of the modules we could do all of this based on an actual example?

Yes, agreed.